Build failure due to ambiguous symbol

I’m seeing a build failure with ghc-8.8.1 and current `hie-bios` (0.3.0) & `ghcide` (6cf1d60d8a763e07fe778695faf5e5c423cc1fcc):

```
$ cabal update
$ cabal install exe:ghcide
…
In order, the following will be built (use -v for more details):
…
 - hie-bios-0.3.0 (lib) (requires download & build)
…
exe/Main.hs:116:78: error:
    Ambiguous occurrence ‘Info’
    It could refer to
       either ‘Development.IDE.Types.Logger.Info’,
              imported from ‘Development.IDE.Types.Logger’ at exe/Main.hs:27:1-35
           or ‘Development.Shake.Info’,
              imported from ‘Development.Shake’ at exe/Main.hs:41:1-37
              (and originally defined in ‘shake-0.18.4:Development.Shake.Internal.Options’)
    |
116 |         ide <- initialise mainRule (pure $ IdInt 0) (showEvent lock) (logger Info) (defaultIdeOptions $ return $ return . grab) vfs
    |                                                                              ^^^^
```

Easy enough to work around locally:

```diff
-        ide <- initialise mainRule (pure $ IdInt 0) (showEvent lock) (logger Info) (defaultIdeOptions $ return $ return . grab) vfs
+        ide <- initialise mainRule (pure $ IdInt 0) (showEvent lock) (logger Development.IDE.Types.Logger.Info) (defaultIdeOptions $ return $ return . grab) vfs
```

Originally encountered by @aymannadeem & myself.

--- a/exe/Main.hs
+++ b/exe/Main.hs
@@ -12,6 +12,7 @@ import System.FilePath
 import Control.Concurrent.Extra
 import Control.Exception
 import Control.Monad.Extra
+import Control.Monad.IO.Class
 import Data.Default
 import System.Time.Extra
 import Development.IDE.Core.FileStore
@@ -38,7 +39,7 @@ import System.Environment
 import System.IO
 import System.Exit
 import Paths_ghcide
-import Development.Shake hiding (Env)
+import Development.Shake (Action, action)
 import qualified Data.Set as Set
 import qualified Data.Map.Strict as Map
 

